slave/components/parser/parser_test.go

```diff
@@ -1,65 +1,2 @@
 package parser
 
-import (
```

Comment:
I do not think we should delete the tests. Yes, they are primitive, but they are very simple and do not provide much value at this moment because the parser was not implemented. To the contrary, I think we should start adding more tests to this file, since the parser is now being implemented. Deleting tests completely, instead of removing independent ones or enriching the test cases, does not sound like good practice to me. I welcome a contrary view though!

Reply:
I did not mean to delete the tests. They were throwing somes errors and therefore I had temporarily removed them, I meant to add them back. I agree. Tests should be present when their respective functions are present. I will add more tests soon before implementing the parser function.